tools/UpdatePaths/__main__.py

```diff
@@ -148,7 +146,7 @@ def get_result(element):
     keys = list(dmm_data.dictionary.keys())
     for definition_key in keys:
         def_value = dmm_data.dictionary[definition_key]
-        new_value = tuple(y for x in def_value for y in get_result(x) if y is not None)
+        new_value = tuple(y for x in def_value for y in get_result(x) if y != None)
```

Comment:
```suggestion
        new_value = tuple(y for x in def_value for y in get_result(x) if y is not None)
```

Bad practice could yield unexpected results